protected/humhub/modules/admin/jobs/DisableModuleJob.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\admin\jobs;
+
+use Yii;
+use humhub\modules\queue\ActiveJob;
+
+class DisableModuleJob extends ActiveJob
```

Comment:
We have a feature called `ExclusiveJob` this can be used to ensure that only one job of the same type (e.g. `DisableModule-polls`) is in the queue once.

https://github.com/humhub/humhub/blob/master/protected/humhub/modules/search/jobs/RebuildIndex.php#L33

It is also possible to check whether such a job is already in the queue. So we could e.g. throw an exception, if someone directly after the Disable of a module, it try to enable it again and the Disable job was not yet executed.

See `QueueHelper::isQueued()`.